### DISCUSSION TITLE
Where to find options for nativePdfFormat and pdfFormat?

### DISCUSSION THREAD

**Discussion by mmahalwy** [gotenberg/gotenberg#536] [Q&A] [answered]
Is there a list somewhere to reference? Could add this to the docs

**Comment by gulien** [comment, marked as answer]
Hello @mmahalwy,

Currently, only the uno PDF engine provides the capability to convert to a specific PDF format. You may find the list of accepted values here: https://gotenberg.dev/docs/modules/pdf-engines#uno
